wp/lib/bap_wp/src/output.ml

```diff
@@ -221,11 +225,14 @@ let output_bildb (solver : Solver.solver) (status : Solver.status) (env : Env.t)
   match status with
   | Solver.SATISFIABLE ->
     info "Outputting BilDB init script to %s\n%!" filename;
-    let module Target = (val target_of_arch (Env.get_arch env)) in
     let model = Constr.get_model_exn solver in
     let mem_model = get_mem model env in
     let var_map = Env.get_var_map env in
-    let reg_map = VarMap.filter_keys ~f:(Target.CPU.is_reg) var_map in
+    let target = (Env.get_arch env) in
```

Comment:
Would it be a good idea to refactor this out so we don't have duplicated code?

Reply:
Agreed.